tasks/compile.py

```diff
@@ -79,6 +78,9 @@ def compile_user(ctx, user, clean=False, debug=False, cp=False):
                 join(dest_folder, "function.wasm"),
             ]
             call(" ".join(cmd), shell=True, cwd=FUNC_BUILD_DIR)
+        if run:
+            _codegen_user(user, debug)
+            _simple_run_user(user, debug)
```

Comment:
I'd rather keep tasks single-purpose and leave it up to the user to chain them if they want (`inv` allows you to chain multiple tasks in a single command). 

Simple runner isn't yet encapsulated in Python though, which thus makes that suggestion impossible. That would be nice and would have same syntax, i.e. `inv run <user> [func]`. This flow would then look like:

```
inv compile-user <my_user> codegen-user <my_user> run <my_user>
```

Reply:
@mfournial this is now possible, with a slight alteration. I've created `run-user` which just takes a user argument, and `run` which takes a function too (i.e. `run-user mpi` and `run mpi hellompi`). This is to avoid having to do something like `run mpi` and `run mpi --func=hellompi`)